docs/API.md

```diff
@@ -995,6 +1001,8 @@ An object with the following properties:
 * `"params"`: See below.
   `start_domain_test` when the *test* was started.
 * `"results"`: A list of [*test result*][Test result] objects.
+* `"testcase_descriptions"`: A map with the test case IDs as keys and the
+  translated descriptions of the corresponding the test cases as values.
```

Comment:
I suggest the following to use standardized terms for "Test Case Identifier" and "Test Case Description" plus adding some references:
```
* `"testcase_descriptions"`: A map with the [Test Case Identifiers] as keys and the
  translated *Test Case Description* of the corresponding [test cases] as values.

[test cases]: https://github.com/zonemaster/zonemaster/tree/master/docs/specifications/tests#list-of-defined-test-cases
[Test Case Identifiers]: https://github.com/zonemaster/zonemaster/blob/master/docs/internal-documentation/templates/specifications/tests/TestCaseIdentifierSpecification.md
```

Reply:
Done